platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/properties/PropertiesFileReaderTest.java

```diff
@@ -0,0 +1,172 @@
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util.properties;
+
+import static java.lang.String.format;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class PropertiesFileReaderTest {
+    @ClassRule
+    public static TemporaryFolder testFolder = new TemporaryFolder();
+
+    private static final Path NONEXISTENT_PATH = Paths.get("this", "path");
+
+    private static final int VALUES_PER_FILE = 4;
+
+    private static String propertiesTestDirectoryPath;
+
+    private static String propertiesTestEmptyDirectoryPath;
+
+    private static String propertiesTestSingleFilePath;
+
+    private PropertiesFileReader reader = new PropertiesFileReader();
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        File propertiesEmptyDirectory = testFolder.newFolder("empty");
+        File propertiesTestDirectory = testFolder.newFolder("properties");
+
+        propertiesTestEmptyDirectoryPath = propertiesEmptyDirectory.getPath();
+        propertiesTestDirectoryPath = propertiesTestDirectory.getPath();
+        propertiesTestSingleFilePath = Paths.get(propertiesTestDirectoryPath, "test1.properties")
+                .toString();
+
+        File propsFile1 = new File(propertiesTestDirectory, "test1.properties");
+        File propsFile2 = new File(propertiesTestDirectory, "test2.properties");
+        File propsFile3 = new File(propertiesTestDirectory, "test3.properties");
+
+        File randomFile = new File(propertiesTestDirectory, "random.cfg");
+        assert randomFile.createNewFile();
+
+        generateAndSaveProperties(propsFile1, "test1value", VALUES_PER_FILE);
+        generateAndSaveProperties(propsFile2, "test2value", VALUES_PER_FILE);
+        generateAndSaveProperties(propsFile3, "test3value", VALUES_PER_FILE);
+
+    }
+
+    @Test
+    public void testLoadDirectoryNullPath() throws Exception {
+        List resultsDirectory = reader.loadPropertiesFilesInDirectory(null);
+        assertTrue(resultsDirectory.isEmpty());
```

Comment:
While it may make the code a little uglier, there are reasons for favoring `assertThat` over `assertTrue` See : 
https://codice.atlassian.net/wiki/display/DDF/Unit+Testing
http://junit.sourceforge.net/doc/ReleaseNotes4.4.html